roomify.install

```diff
@@ -318,8 +318,6 @@ function roomify_install_finished(&$install_state) {
   variable_set('addtoany_additional_html_placement', 'before');
   variable_set('addtoany_image', 'a2a_svg_32');
 
-  variable_set('commerce_checkout_completion_message', $message);
```

Comment:
@acrollet the value is set in the hook_variable_info  (roomify_accommodation_booking_variable_info)